hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/generator/GenericRecordFullPayloadGenerator.java

```diff
@@ -43,22 +44,39 @@
  */
 public class GenericRecordFullPayloadGenerator implements Serializable {
 
-  public static final int DEFAULT_PAYLOAD_SIZE = 1024 * 10; // 10 KB
+  /**
+   * 10 KB.
+   */
+  public static final int DEFAULT_PAYLOAD_SIZE = 1024 * 10;
   private static Logger log = LoggerFactory.getLogger(GenericRecordFullPayloadGenerator.class);
   protected final Random random = new Random();
-  // The source schema used to generate a payload
+  /**
+   * The source schema used to generate a payload.
```

Comment:
Why we should change these comment styles for fields?

Reply:
> Why we should change these comment styles for fields?

my bad,  that`s the coding guidelines of Alibaba. 
rolled back already :)